### PR TITLE
Implement change_url

### PR DESCRIPTION
**Pull request by yalh76**
## Solution
- *Implement change_url*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
* An automatic package_check will be launch at https://ci-apps-dev.yunohost.org/, when you add a specific comment to your Pull Request: "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!"*

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

File: scripts/change_url [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+cp ../conf/env.prod "$configfile"
+
+ynh_replace_string --match_string="__REDIS_DB__"  --replace_string="$redis_db"   --target_file="$configfile"
+ynh_replace_string --match_string="__PORT__"      --replace_string="$port"       --target_file="$configfile"
+ynh_replace_string --match_string="__DOMAIN__"    --replace_string="$new_domain" --target_file="$configfile"
+ynh_replace_string --match_string="__DB_USER__"   --replace_string="$db_name"    --target_file="$configfile"
+ynh_replace_string --match_string="__DB_PWD__"    --replace_string="$db_pwd"     --target_file="$configfile"
+ynh_replace_string --match_string="__DB_NAME__"   --replace_string="$db_name"    --target_file="$configfile"
+ynh_replace_string --match_string="__FINALPATH__" --replace_string="$final_path" --target_file="$configfile"
+ynh_replace_string --match_string="__KEY__"       --replace_string="$key"        --target_file="$configfile"
```
Comment (ericgaspar) [inline scripts/change_url:127]:
```
domain=$new_domain
ynh_add_config --template="../conf/env.prod" --destination="$configfile"
```

Is `ynh_add_config` better?

Comment (yalh76) [reply scripts/change_url:127]: yep would be far more better, but I copy paste yours ;)

**Review by ericgaspar — APPROVED** [review]

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
@ericgaspar change_url is ready :)

**Pull request merged**